cmr-exchange/service-bridge/resources/docs/rest-api/index.md

```diff
@@ -723,26 +723,26 @@ or `variables[](or variable_aliases[]`, which is repeated for each variable(or v
 A `service_id` parameter is optional; if not provided, the service type of `opendap` is assumed.
 otherwise, the service type will be retrieved using the service_id through ous call to CMR.
 
-A `format` parameter is optional; if not provided, the format of `nc` (NetCDF3) is assumed for 
+A `format` parameter is optional; if not provided, the format of `nc` (NetCDF3) is assumed for
 `opendap` service type and the format of `native` is assumed for `esi` service type.
-Currently supported `format` values are:
+Currently supported `format` values are(case insensitive):
```

Comment:
Knit-pick: Maybe put a space between are and (case insensitive)

Reply:
Done.